package.json

```diff
@@ -316,6 +318,8 @@
         "yargs": "^15.1.0"
     },
     "peerDependencies": {
+        "@box/blueprint-web": "^7.2.4",
```

Comment:
is there any reason why we have different version in peerDependecies?

Reply:
comparing to https://github.com/box/box-ui-elements/pull/3585/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R126

Reply:
fixed